src/Tribe/Views/V2/Views/Recent_Past_View.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * The Past Recent View.
+ *
+ * @package Tribe\Events\Views\V2\Views
+ * @since TBD
+ */
+
+namespace Tribe\Events\Views\V2\Views;
+
+use Tribe\Events\Views\V2\View;
+use Tribe\Events\Views\V2\Views\Traits\List_Behavior;
+use Tribe__Context;
+use Tribe__Events__Main as TEC;
+use Tribe__Events__Rewrite as TEC_Rewrite;
+use Tribe__Utils__Array as Arr;
+use Tribe\Events\Views\V2\Utils;
+
+class Recent_Past_View extends View {
+	use List_Behavior;
+	/**
+	 * Slug for this view
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected $slug = 'recent-past';
+
+	/**
+	 * Visibility for this view.
+	 *
+	 * @since TBD
+	 *
+	 * @var bool
+	 */
+	protected static $publicly_visible = true;
+
+	/**
+	 * Indicates Recent Past View supports the date as a query argument appended to its URL, not as part of a "pretty" URL.
+	 *
+	 * @var bool
+	 */
+	protected static $date_in_url = false;
+
+		// Todo: remove this.
+
+	public function maybe_filter_current_view_slug( $view_slug, $context, $query ) {
+
```

Comment:
:no_entry_sign: You must use "/**" style comments for a function comment

<sub>hash: 7dffdf6922b77a58ede29385330888a9</sub>